engine/src/main/java/org/camunda/bpm/engine/impl/util/ParseUtil.java

```diff
@@ -116,4 +116,18 @@ public static ProcessEngineDetails parseProcessEngineVersion(String packageImple
 
     return new ProcessEngineDetails(version, edition);
   }
+
+  public static String parseServerVendor(String applicationServerInfo) {
+    String serverVendor = null;
+    Pattern pattern = Pattern.compile("[ A-Za-z]+");
+    Matcher matcher = pattern.matcher(applicationServerInfo);
+    if (matcher.find()) {
+      serverVendor = matcher.group(0).trim();
+    }
+    if (serverVendor.contains("WildFly")) {
+      return "JBoss WildFly";
```

Comment:
```suggestion
      return "WildFly";
```
I think the official name is not `JBoss Wildfly`. What is the idea behind this wording?